src/calliope/schemas/config_schema.py

```diff
@@ -19,15 +19,29 @@
 ]
 
 
+class InitMath(CalliopeBaseModel):
+    """Defines the math to add to the model.
+
+    Files will be loaded in the following order: base -> mode -> extra.
+    """
+
+    base: AttrStr = "plan"
+    """Name of the math file to build on top of.
+    Can be any pre-defined math file or user-defined `extra` file.
+    """
+    extra: dict[AttrStr, str] = Field(default={})
+    "Dictionary with the names and paths of additional math files."
```

Comment:
In an effort to fail fast, could we have a validation step which checks whether `base` is either an extra key or a pre-defined math name?